modules/ui/src/app/pages/risk-assessment/profile-item/profile-item.component.html

```diff
@@ -14,17 +14,30 @@
 limitations under the License.
 -->
 <div class="profile-item-container">
-  <mat-icon class="profile-item-icon" fontSet="material-symbols-outlined">
-    rule
+  <mat-icon
+    *ngIf="profile.status === ProfileStatus.VALID"
+    class="profile-item-icon"
+    fontSet="material-symbols-outlined">
+    check_circle
   </mat-icon>
-  <p
-    class="profile-item-name"
-    role="button"
-    tabindex="0"
-    (click)="profileClicked.emit(profile)"
-    (keydown.enter)="profileClicked.emit(profile)">
-    {{ profile.name }}
-  </p>
+  <mat-icon
+    *ngIf="profile.status === ProfileStatus.DRAFT"
```

Comment:
There is also can be the value as "Expired" that has different icon.
The link to the doc: https://docs.google.com/document/d/1XI-Jv-tEHRc9x9TOy9SBKixYSai6i9YmZYm0LFzkVv4/edit?resourcekey=0-LywLMRKxfE-5Ax9R0QbeaQ&tab=t.0#heading=h.94byhq6mp4bc

Should we also add it or there is another ticket for it?

Reply:
It'll be added later